happi/item.py

```diff
@@ -335,6 +335,7 @@ def show_info(self, handle=sys.stdout):
         """
 
         pt = PrettyTable(['EntryInfo', 'Value'])
+        pt._max_width = {'Value': 60}
```

Comment:
This is the width of the full table?

I think a max width is appropriate and 60 is reasonable, but I wonder if we should also think about getting the user's terminal width to help us set this properly.

Reply:
Also: is there some better way to do this than to use a semi-private variable? Just `max_width` maybe?

Reply:
You can set the max width of the whole table via `pt.max_table_width` and the width by column via `pt._max_width`

Getting the user's terminal width would be cool, I didn't think of that.  It looks easy enough to do too.

Reply:
It looks like there is also a `max_width` property that reads from `_max_width`- might want to avoid poking at the internals if we don't have to.

Reply:
Also `max_width` will also steal space from the other columns, which I didn't want.  I'll see if I can't refine this more